src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/ClientBaseTests.cs

```diff
@@ -219,6 +219,93 @@ MessageHeader header
         }
     }
 
+    [Fact]
+    [OuterLoop]
+    public static void DefaultSettings_Echo_Cookie()
+    {
+        // *** SETUP *** \\
+        var factory = new ChannelFactory<IWcfAspNetCompatibleService>(
+                 new BasicHttpBinding()
+                 {
+                     AllowCookies = true
+                 },
+                 new EndpointAddress(Endpoints.HttpBaseAddress_Basic));
+
+        IWcfAspNetCompatibleService serviceProxy = factory.CreateChannel();
+
+        IHttpCookieContainerManager cookieManager = ((IChannel)serviceProxy).GetProperty<IHttpCookieContainerManager>();
+        Assert.True(cookieManager != null);
+        Assert.True(cookieManager.CookieContainer != null);
+        string cookieName = "cookieName";
+        string cookieValue = "cookieValue";
+        string cookieSentOut = $"{cookieName}={cookieValue}";
```

Comment:
Please double check that the C# 6 features are enabled and working for all platforms; a couple of months ago, we ran into an issue when compiling x-plat tests/ToF (I don't remember which now...) where C#6 features wouldn't work and caused breaks